en: improve head documentation (#1659)
nuxt/docs: en/api/pages-head.md

Use the `head` method to set the HTML Head tags for the current page.

> Your component data are available with `this` in the `head` method, you can use set custom meta tags with the page data. Be sure to also have a look in the Nuxt Faqs the page about [external resources](https://nuxtjs.org/faq/).

Reviewer: ```suggestion
Your component data are available with `this` in the `head` method, you can use set custom meta tags with the page data. Be sure to also take a look at the [Nuxt FAQ](https://nuxtjs.org/faq/).
```
